docs/user/dashboard/lens.asciidoc

```diff
@@ -220,3 +227,49 @@ Visualizing saved searches in unsupported in *Lens*.
 ===== How do I change the number of suggestions?
 
 Configuring the *Suggestions* that *Lens* automatically populates is unsupported.
+
+[float]
+[[is-it-possible-to-use-different-indexpatterns-in-lens]]
+===== Is it possible to use different index patterns in a single visualization?
+
+You can create a visualization from multiple index patterns in *Lens*: this feature is available for a subset of visualizations such as *Bar*, *Line* and *Area* charts.
+
+Each *Layer* in a visualization is associated with an index pattern and mutiple *Layers* can be combined together within the same visualization. Each *Layer* also has a chart switcher button in order to select the best type of visualization for the specific dataset.
```

Comment:
```suggestion
Each chart *Layer* is associated with an index pattern, and you can combine multiple *Layers* in the same chart. Each *Layer* also has a menu to change the visualization type, which allows you to select the best type of visualization for the data.
```

Reply:
Since this section is about using multiple index patterns, is this sentence necessary?

Reply:
I think that, while it may sounds a distinct issue, there is often a correlation between overlaying different charts types from different index patterns.